modules/gapi/test/render/gapi_render_tests_ocv.cpp

```diff
@@ -418,6 +474,25 @@ INSTANTIATE_TEST_CASE_P(RenderNV12OCVTestTextsImpl, RenderNV12OCVTestTexts,
                                 Values(LINE_8),
                                 Values(false)));
 
+#ifdef HAVE_FREETYPE
+
+INSTANTIATE_TEST_CASE_P(RenderBGROCVTestFTextsImpl, RenderBGROCVTestFTexts,
```

Comment:
Looks like these tests are not available (disabled by `#ifdef HAVE_FREETYPE`) on current Linux builders.
Also there is no "HAVE_FREETYPE" in compilation logs.

For testing, font has been installed on `linux-1` buildworker's `ubuntu:16.04` image (`Linux OpenCL` is scheduled there)

Reply:
Please verify "Custom" builder results - should be fixed there.